main.js

```diff
@@ -26,3 +26,79 @@ window.addEventListener('scroll', () => {
     .querySelector('nav')
     .classList.toggle('screen-scroll', window.scrollY > 0);
 });
+
+const projects = [
+  {
+    name: 'Multi-post stories',
+    image: 'img/a.jpg',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text",
+    technologies: ['Html', 'Css', 'Bootstrap', 'Ruby'],
+    liveLink: 'https://josphat205.github.io/SmartDeveloper/',
+    githubLink: 'https://github.com/Josphat205/SmartDeveloper',
+  },
+  {
+    name: 'Multi-post stories',
+    image: 'img/b.jpg',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text",
+    technologies: ['Html', 'Css', 'Bootstrap', 'Ruby'],
+    liveLink: 'https://josphat205.github.io/SmartDeveloper/',
+    githubLink: 'https://github.com/Josphat205/SmartDeveloper',
+  },
+  {
+    name: 'Multi-post stories',
+    image: 'img/c.jpg',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text",
+    technologies: ['Html', 'Css', 'Bootstrap', 'Ruby'],
+    liveLink: 'https://josphat205.github.io/SmartDeveloper/',
+    githubLink: 'https://github.com/Josphat205/SmartDeveloper',
+  },
+  {
+    name: 'Multi-post stories',
+    image: 'img/d.jpg',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text",
+    technologies: ['Html', 'Css', 'Bootstrap', 'Ruby'],
+    liveLink: 'https://josphat205.github.io/SmartDeveloper/',
+    githubLink: 'https://github.com/Josphat205/SmartDeveloper',
+  },
+];
+
+const demo = document.getElementById('demo');
+demo.innerHTML = projects.reduce((output, project) => (
+  `${output
+  }
```

Comment:
- [ ] For mobile screens please reduce the top margin for the popup model, Right now complete model is not visible on the screen, and make sure it matches with the Figma design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/167914055-f3c644ea-93e6-4aec-ac27-fff8bd2f8c74.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/167914092-0738a31c-67ab-4364-ac74-4dcd0c4d7730.png)